pythonforandroid/recipes/android/__init__.py

```diff
@@ -1,23 +1,24 @@
-from pythonforandroid.recipe import CythonRecipe, IncludedFilesBehaviour
+from pythonforandroid.recipe import PyProjectRecipe, IncludedFilesBehaviour
 from pythonforandroid.util import current_directory
 from pythonforandroid import logger
 
 from os.path import join
 
 
-class AndroidRecipe(IncludedFilesBehaviour, CythonRecipe):
+class AndroidRecipe(IncludedFilesBehaviour, PyProjectRecipe):
     # name = 'android'
     version = None
     url = None
 
     src_filename = 'src'
 
     depends = [('sdl3', 'sdl2', 'genericndkbuild'), 'pyjnius']
+    hostpython_prerequisites = ["Cython>=3.1,<3.2"]
```

Comment:
Agr my bad, I didn't realise the Cython 3.1 build was failing on some arch e.g. x86
I played with it a bit and I got it working with Cython 0.29 and 3.0 with x86, so I guess we should change it to:
```
hostpython_prerequisites = ["Cython>=0.29,<3.1"]
```
Sorry for misleading you with my initial comment thinking 3.1 would do it

Reply:
Thanks for the quick fixes and the neat rebasing as always :muscle: